src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$--main-color: #fff;
+$--code-color: #616070;
+$--button-color: #00acdc;
+$--text-color: #060b35;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)